src/styles/index.scss

```diff
@@ -1,3 +1,139 @@
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$button-color: #00acdc;
+$button-text-color: #fff;
+
 body {
+  font-family: Roboto, sans-serif;
+  padding: 0;
   margin: 0;
 }
+
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: start;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 198px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 0 0;
+  }
+
+  &__tittle {
+    margin: 40px 16px 0;
+
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    margin: 4px 16px 0;
+
+    color: $secondary-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 16px;
+    width: 166px;
+    height: 40px;
+    background: $button-color;
+    border-radius: 5px;
+
+    text-align: center;
+    color: $button-text-color;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    &:hover {
+      color: $button-color;
+      background-color: $button-text-color;
+      border: 1px solid $button-color;
+    }
+  }
+}
+
+.reviews {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 166px;
+
+  color: $main-text-color;
+
+  &__stars {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 16px 0 0;
+  }
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+
+    width: 16px;
+    height: 16px;
+
+    &:nth-child(-n+4) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &__number {
+    display: flex;
+    align-items: center;
+    color: $main-text-color;
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
`font-weight: 400;` is a default browser value. Remove from similar cases as well
```suggestion
```